### PR TITLE
Create cutyoururl.ashishagarwal.json

### PR DESCRIPTION
**Pull request by devashish2024**
> Actually it redirects to PythonAnywhere... thanks

Please note that we reserve the rights not to accept any domain at our own discretion.

- [X] The file is in the `domains` folder and is in the JSON format.
- [X] You have completed your website. 
- [X] The website is reachable.  
- [X] You're not using Vercel or Netlify.  
- [X] The CNAME record doesn't contain `https://` or `/`.  
- [X] There is sufficient information at the `owner` field.

### CODE REVIEW

**Comment by ashishagarwal2023** [comment]
Btw how can I get a email? I'd like to get hello@ashish.is-a.dev, can I? What will be its limitations and features? Login with  gmail?

**Comment by ashishagarwal2023** [comment]
Can anyone respond bruhv

File: domains/cutyoururl.ashishagarwal.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "CutYourURL is a free URL Shortener!",
+  "repo": "https://github.com/ashishagarwal203/CutYourURL",
+  "owner": {
+    "username": "ashishagarwal2023",
+    "twitter": "",
+    "email": "code.with.aasheesh@gmail.com"
```
Comment (CuteDog5695) [inline domains/cutyoururl.ashishagarwal.json:7]: if you dont want to list your twitter then remove the twitter line

File: domains/cutyoururl.ashishagarwal.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  },
+  "record": {
+    "CNAME": "ashishagarwal.is-a.dev/cutyoururl"
```
Comment (CuteDog5695) [inline domains/cutyoururl.ashishagarwal.json:10]: CNAME cant have / and anything after that (also site gives 404 page)

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by CuteDog5695** [comment]
> Btw how can I get a email? I'd like to get [hello@ashish.is-a.dev](mailto:hello@ashish.is-a.dev), can I? What will be its limitations and features? Login with gmail?

you can use improvmx which is free (tho its a email forwarding service so you cant send emails with it)

**Comment by ashishagarwal2023** [comment]
@CuteDog5695 ok just asking theres no official .is-a.dev's free emails?
ACCEPT PR BRU

File: domains/cutyoururl.ashishagarwal.json (hunk 1 above)
Comment (ashishagarwal2023) [reply domains/cutyoururl.ashishagarwal.json:7]: ok

File: domains/cutyoururl.ashishagarwal.json (hunk 2 above)
Comment (ashishagarwal2023) [reply domains/cutyoururl.ashishagarwal.json:10]: https://ashishagarwal.is-a.dev/CutYourURL/
this was what i meant, but bruh ur response made me struggle...

**Pull request closed without merging**